14/umbraco-forms/tutorials/creating-a-multipage-form.md

```diff
@@ -0,0 +1,161 @@
+
+For this tutorial, the following fields are added with the respective specifications:
+
+| Field Name    | Data Type    | Field Type |
+|---------------|--------------|------------|
+| Surname       | Short answer | text       |
+| Age           | Short answer | number     |
+| Country       | Short answer | text       |
+| Phone number  | Short answer | tel        |
+| Email address | Short answer | email      |
+
+If you wish to reorder your fields, click **Reorder**.
+
+![First Page of the Form](images/first-page-of-form.png)
+
+## Create the Second Page
+
+To create a multi-page form, you need to add more pages:
+
+1. Click **Add new page** to create the second page of your form.
+2. **Enter a Name** for this page. For example: *Company Information*.
+3. Click **Add question**.
+4. Select **Short answer** as the field type.
+5. **Enter a Name** for the field type. For example, *Company Name*.
+6. Provide a **Default Value**. For example, *Enter the name of your company*.
+7. Click **Submit**.
+8. Click **Save**.
+
+![Company information](images/company-information.png)
+
+Similarly, you can also add other relevant fields based on your requirements.
+
+### Add Conditional Logic
+
+Umbraco Forms allows you to customize the flow of your multi-page form. You can add conditional logic to control which questions appear based on user inputs.
+
+To add conditions, follow these steps:
+
+1. Click **Add question**.
+2. Select **Single choice** as the field type.
+3. **Enter a Name** for the field type. For example, *Do you work with Umbraco*.
+4. Enter the **Value** and **Caption** in the **Options** field.
+
+![Options Field](images/prevalue-fields.png)
+
+5. Mark the field as **Mandatory**.
+6. Click **Submit**.
+7. Click **Save**.
+8. Repeat steps 1-4 to create a conditional question titled *If yes, how many years?*
+
+![Next Question](images/conditional-question-part-1.png)
+
+9. Enable **Conditions**.
+10. Set the parameters for the condition as follows:
+
+![Add condition](images/conditional-question-part-2.png)
+
+11. Click **Submit**.
+12. Click **Save**.
+
+![Second Page Details](images/page-2-details.png)
+
+## Create the Final Page
+
+1. Click **Add new page** to create the final page of your form.
+2. **Enter a Name** for this page. For example: *Products*.
+3. Click **Add question**.
+4. Select **Multiple choice** as the field type.
+5. **Enter a Name** for the field type. For example, *Select the products you are interested in*.
+6. Enter the **Value** and **Caption** in the **Options** field.
+
+![Question Options](images/multiple-choice.png)
+
+7. Click **Submit**.
+8. Click **Save**.
+
+![Final Page Details](images/Final-page.png)
+
+## Embed the Form on a Web Page
+
+Once you are satisfied with your multi-page form, it is time to embed it on your website.
+
+To display the form on the website, follow these steps:
+
+1. Navigate to the **Content** section of Umbraco.
+2. Click **+** next to the parent page of the website.
```

Comment:
Is a step for picking a Document Type missing here?
Suggestion: Perhaps there should be a prerequisite to this tutorial, for having a Document Type with the Form Picker?

Reply:
Oh no 🙈, Yes a step and an image is missing 😳. I like the idea of adding a Prerequisite section as this section expects users to have some existing content to embed the Form.